### ISSUE TITLE
Problem: Pending transactions are not represented on transactions page.

### ISSUE THREAD

**Issue by n1tranquilla** [Granola-Team/mina-block-explorer#254]

**Comment by n1tranquilla** [comment]
On hold for now, as pending transactions require exposing a new API endpoint for the node/cluster of nodes.

**Comment by trevorbernard** [comment]
In order to display pending transactions in the frontend we must have access to a Mina node so we may query the mempool. Until we have such a API, this will remain on hold.

**Comment by n1tranquilla** [comment]
We will use reverse proxy to unblock ourselves for now until we have our own node setup.

**Comment by jhult** [comment]
### Proxies to a Mina node

- https://mina-mainnet-graphql.aurowallet.com/graphql
- https://proxy.minaexplorer.com/graphql

If the above have issues with CORS, let me know and I can help setup a reverse proxy.

### GraphQL Query
```
query pendingTx($address: PublicKey!) {
  pooledUserCommands(publicKey: $address) {
		id
		hash
		kind
		nonce
		source {
		  publicKey
		}
		receiver {
		  publicKey
		}
		amount
		fee
		memo
		failureReason
		feeToken
  }
}

```

### GraphQL Variables

You can find pending transactions and a public key to use [here](https://minaexplorer.com/pending-transactions).

```
{
  "address": "B62qov9yv8TayLteD6SDXvxyYtmn3KkUoozAbs47fVo9JZSpcynbzTz"
}
```

**Comment by robinbb** [comment]
@n1tranquilla Why am I assigned? If it is to get our own node configured, so that we can query it directly, as opposed to the minaexplorer.com proxy, then let's use this issue for configuring the proxy and getting the features into the front-end.

**Comment by n1tranquilla** [comment]
Assigned for review. This feature is done.

**Comment by jhult** [comment]
I believe this was implemented in https://github.com/Granola-Team/mina-block-explorer/pull/440.
